docroot/modules/custom/mass_unpublish_reminders/src/Plugin/QueueWorker/UnpublishReminderQueue.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace Drupal\mass_unpublish_reminders\Plugin\QueueWorker;
+
+use Drupal\Core\Queue\QueueWorkerBase;
+use Drupal\node\Entity\Node;
+use Drupal\Core\Url;
+
+/**
+ * Processes Tasks for mass_unpublish_reminders.
+ *
+ * @QueueWorker(
+ *   id = "mass_unpublish_reminders_queue",
+ *   title = @Translation("Node Unpublish Reminders: email queue"),
+ *   cron = {"time" = 60}
+ * )
+ */
+class UnpublishReminderQueue extends QueueWorkerBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function processItem($nid) {
+    $node = Node::load($nid);
+    $author = $node->getOwner();
+    if (!empty($author)) {
+      $author_mail = $author->getEmail();
+      $params = [];
+      $cc_mails = [];
+      $unpublish_date = '';
+      $url = '';
```

Comment:
@arthurbaghdas This appears to be an unused variable. We should remove it if that is the case.

Reply:
Refactored the code to remove those variables.